.github/workflows/test.yml

```diff
@@ -1,23 +1,14 @@
 ---
-#based on https://raw.githubusercontent.com/ckan/ckanext-scheming/master/.github/workflows/test.yml
-# alternative https://github.com/ckan/ckan/blob/master/contrib/cookiecutter/ckan_extension/%7B%7Bcookiecutter.project%7D%7D/.github/workflows/test.yml
 name: Tests
 on: [push, pull_request]
-env:
-  CKAN_SQLALCHEMY_URL: postgresql://ckan_default:pass@postgres/ckan_test
-  CKAN_DATASTORE_WRITE_URL: postgresql://datastore_write:pass@postgres/datastore_test
-  CKAN_DATASTORE_READ_URL: postgresql://datastore_read:pass@postgres/datastore_test
-  CKAN_SOLR_URL: http://solr:8983/solr/ckan
-  CKAN_REDIS_URL: redis://redis:6379/1
-
 jobs:
   lint:
     runs-on: ubuntu-18.04
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
         with:
-          python-version: '2.7'
+          python-version: '3.6'
```

Comment:
It could be, but generally Python 3 is stricter than Python 2, so it's not been a problem thus far. It would become an issue if we weren't using `six` as much as possible.